usersimcrs/simulator/agenda_based/agenda_based_simulator.py

```diff
@@ -58,6 +59,41 @@ def _generate_response(self, agent_utterance: Utterance) -> Utterance:
         """
         return self.generate_response(agent_utterance)
 
+    def _generate_item_preference_response_intent(self, item_id: str) -> Intent:
+        """Generates response preference intent for a given item id.
+
+        Args:
+            item_id: Item id.
+
+        Returns:
+            Preference intent.
+        """
+        if item_id is None:
+            # The recommended item was not found in the item collection.
+            response_intent = self._interaction_model.INTENT_DONT_KNOW  # type: ignore[attr-defined] # noqa
```

Comment:
You can just return INTENT_DONT_KNOW here, then the rest of the method does not need to be nested inside an else.

Reply:
Done.